packages/web-app-files/src/mixins/actions/downloadArchive.js

```diff
@@ -21,6 +21,7 @@ export default {
           },
           isEnabled: ({ resources }) => {
             if (
+              this.isFilesAppActive &&
```

Comment:
Could you please add a comment here (and in the other mixins where this check is used) that we only exit for routes of the files app, because we don't want to include knowledge about other apps within the files app. Eventually we'll want to refactor the mixins into something nicer + typescript-ier and probably also want to allow registration of context actions via the extension system. But we're not there, yet.

Reply:
Added comment to the `useIsFilesAppActive` composable.